redirects/7.0/concepts/OOoJodconverter-subsystem-props.html

```diff
@@ -0,0 +1 @@
+/content-services/7.0/config/libreoffice/
```

Comment:
Adding a note, so this one (like the 7.1 version) is changed.

Reply:
Thanks @Amy-Hyland  - this is tracked by DOCS-6849.